includes.php

```diff
@@ -66,6 +66,191 @@
 		</header>
 		<main>';
 
+$mysqli = new mysqli( 'localhost', 'patchdemo', 'patchdemo', 'patchdemo' );
+if ( $mysqli->connect_error ) {
+	die( $mysqli->connect_error );
+}
+
+function insert_wiki_data( string $wiki, string $creator, int $created, array $patches = [] ) {
+	global $mysqli;
+	$stmt = $mysqli->prepare( '
+		INSERT INTO wikis
+		(wiki, creator, created, patches)
+		VALUES(?, ?, NOW(), ?)
+		ON DUPLICATE KEY UPDATE
```

Comment:
I don't like this but I think I'll propose my own change. (This looks like a good way to lose lots of data and in particular to reset the `created` date accidentally; but it looks like it needs a few changes elsewhere.)

Reply:
I see now that it only updates `patches`? (next line)

I missed it at first so that must mean it's very confusing. I still don't like it. :)

Reply:
Yeah - we should use some database abstraction framework probably. Writing queries gets out of hand fast.